### PR TITLE
Create about.md

### PR DESCRIPTION
**Pull request by esparkhu**

### CODE REVIEW

**Comment by Artemis21** [comment]
This seems largely to duplicate `index.md` (wiki home)?

**Comment by Artemis21** [comment]
Also you've used `*` for bullets in some places but `-` in others, harmless but why?

**Comment by esparkhu** [comment]
Yes, It is pretty much the same as the wiki home. I don't know what else to put on the about page. Do we even need it? I'm open to suggestions.

I used - instead of bullets because I was copy/pasting from discord. It wasn't a conscious choice.

**Review by Artemis21 — CHANGES_REQUESTED** [review]
Just minor link formatting issues.

Edit: Looks like my review wasn't attached, will do another one.

File: content/about.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@title About
+@navbar
+
+# The Summer Polympics 
+
+The summer Polympics is an offically sponsored, multi-event Polytopia tournament inspired by the real Olympics. Teams of up to 12 players will be formed based on nationality (or region in the case of very large nations like the US). Teams will send players to participate in a variety of event contests (single player, multiplayer, multiplayer team, and even art events). Gold, silver and bronze medals will be awared for each event. Gold medal winners will also be awarded one month nitro gifts as prizes.
+
+Players register)https://polytopia.fun/login) and join a team based on nationality. Participants will need a discord account. Teams will send players to compete in ten different events. Some events allow multiple entries per team, others are limited to one representative (or pair of representatives) per team. See events (https://polytopia.fun/events) rules for more details. Some events are mobile only while others allow for Steam or mobile users.
```
Comment (Artemis21) [inline content/about.md]:
```suggestion
Players [register](/login) and join a team based on nationality. Participants will need a discord account. Teams will send players to compete in ten different events. Some events allow multiple entries per team, others are limited to one representative (or pair of representatives) per team. See [event rules](/wiki/events) for more details. Some events are mobile only while others allow for Steam or mobile users.
```

**Review by Artemis21 — CHANGES_REQUESTED** [review]
As above.

**Pull request closed without merging**